src/main/java/uk/gov/hmcts/reform/sscs/model/ReferenceData.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.sscs.exception.ReferenceDataImportException;
+import uk.gov.hmcts.reform.sscs.utility.JsonDataReader;
+
+@Slf4j
+public abstract class ReferenceData {
+
+    protected static <T> List<T> getReferenceData(String filename, TypeReference<List<T>> typeReference) {
+        try {
+            return JsonDataReader.importObjectDataList(filename,  typeReference);
+        } catch (IOException exception) {
+            String message = String.format("Error setting Reference Data while reading from %s, %s", filename, exception.getMessage());

```

Comment:
Very minor: Change wording here to "Error loading" and this variable is not strictly needed, I would just place it inline with log.error below.